src/services/auth-service.ts

```diff
@@ -0,0 +1,44 @@
+import { initializeApp } from 'firebase/app';
+import { Auth, getAuth, signInWithEmailAndPassword } from "firebase/auth";
+import { getEnvVar } from "../helpers/envVars";
+import userProps from "../types/UserProps";
+
+export class AuthService {
+    private static apiKey?: string
+    private static authDomain?: string
+    private static databaseURL?: string
+    static projectId?: string
+    private static storageBucket?: string
+    private static messagingSenderId?: string
+    private static appId?: string
```

Comment:
I believe this static props might be unnecessary since they are only used inside the `initialize` function. You can leave it as is, this is a bit 'picky' 😝 

Just so you have it in mind for next time. It's nice to prepare your code for the future, but be careful falling into the "what ifs" whole, its very dark in there

Reply:
Actually this is a copy paste from the backend, and it was developed by Charlie, so maybe we have to tell him, hahaha

I totally agree with you, those statics are not necessary 👍 I'll write it down to change that in the frontend and backend AuthService in another issue. Thank you!

Reply:
Now that we are talking about statics, do you use them inside the class methods with `this.` or with `AuthService.` because I realise the other day that you can call them with both and now I don't know which one is better